courses/all_course_model.h

```diff
@@ -10,26 +10,26 @@
  * in this file or problems will arise in other parts of the code
  **/
 
-extern mk64_Vtx d_course_big_donut_vertex[0x48D];
-extern mk64_Vtx d_course_block_fort_vertex[0x440];
-extern mk64_Vtx d_course_double_deck_vertex[0x22B];
-extern mk64_Vtx d_course_skyscraper_vertex[0x43E];
-extern mk64_Vtx d_course_choco_mountain_vertex[0x15B8];
-extern mk64_Vtx d_course_frappe_snowland_vertex[0x1599];
-extern mk64_Vtx d_course_mario_raceway_vertex[0x167D];
-extern mk64_Vtx d_course_toads_turnpike_vertex[0x18D7];
-extern mk64_Vtx d_course_kalimari_desert_vertex[0x18F9];
-extern mk64_Vtx d_course_koopa_troopa_beach_vertex[0x24A0];
-extern mk64_Vtx d_course_luigi_raceway_vertex[0x1730];
-extern mk64_Vtx d_course_moo_moo_farm_vertex[0x1F24];
-extern mk64_Vtx d_course_banshee_boardwalk_vertex[0x1351];
-extern mk64_Vtx d_course_dks_jungle_parkway_vertex[0x162F];
-extern mk64_Vtx d_course_rainbow_road_vertex[0xC27];
-extern mk64_Vtx d_course_yoshi_valley_vertex[0xE88];
-extern mk64_Vtx d_course_bowsers_castle_vertex[0x2537];
-extern mk64_Vtx d_course_royal_raceway_vertex[0x2072];
-extern mk64_Vtx d_course_sherbet_land_vertex[0xA76];
-extern mk64_Vtx d_course_wario_stadium_vertex[0x17B3];
+extern Course_Vtx d_course_big_donut_vertex[0x48D];
```

Comment:
Can we do CourseVtx instead? Struct should be CamelCase with no underlines. We were just breaking the rules with the underline.